index.ipynb

```diff
@@ -0,0 +1,283 @@
+{
```

Comment:
<ul><li>It's kind of hard to follow the different settings in the opening paragraph. Could you add a table describing the different simulations?</li><li>Colorbar on plots. Are they the same scale?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/sitcomtn-085/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>added a table and colorbars to each plots.  Previously they were set by <code>ZScaleInterval()</code> and were not identical (although within few counts)  for each intra/extra pairs.  </p>